Source/WebKit/WPEPlatform/wpe/WPESettings.cpp

```diff
@@ -555,6 +558,47 @@ WPE_SETTINGS_HELPER_API(const char*, string, G_VARIANT_TYPE_STRING, nullptr)
  * the return type.
  */
 WPE_SETTINGS_HELPER_API(gdouble, double, G_VARIANT_TYPE_DOUBLE, 0.0)
+/**
+ * wpe_settings_set_uint32_pair:
+ * @settings: a #WPESettings
+ * @key: the key to look up
+ * @value: the value to set
+ * @source: the source of the settings change
+ * @error: return location for a #GError or %NULL
+ *
+ * This is a simple wrapper around [method@WPESettings.set_value] that also validates
```

Comment:
I don't think we want a wrapper for this, for complex types we can just use set_value/get_value